settings/default/login.lua

```diff
@@ -36,4 +36,25 @@ xi.settings.login =
 
     -- Number of simultaneous game sessions per IP (0 for no limit)
     LOGIN_LIMIT = 0,
+
+    -- If true, blocks character creation with names of NPCs and Mobs in the database (Fafnir, Shantotto, etc.)
+    DISABLE_MOB_NPC_CHAR_NAMES = false,
+
+    -- Character names with any of these words in, in any position, will be rejected
+    --
+    -- Examples that will be rejected (using "badword"):
+    -- "badword"
+    -- "imbadword"
+    -- "badwordisme"
+    -- "lolbadwordlol"
+    --
+    -- WARNING:
+    -- Be aware of the "Scunthorpe problem"!
```

Comment:
For this reason, you may want to have two lists:
- wildcard banned names
- full word banned names

Some banned words are tricky to wildcard, especially the shorter ones. Banning the exact word as a name (as you do with NPCs) could be an alternative.

Reply:
I'm going to leave it as it is, since wildcard and whole-word are handled by the list. I think in terms of `the Scunthorpe problem`, you can have the offending word in many offensive configurations, but many less legitimate ones. Especially in terms of player names. I think it's totally OK to get some false positives if it means all bad words are blocked